dashboard/test/ui/features/step_definitions/pd.rb

```diff
@@ -1,3 +1,13 @@
+Given(/^I am a workshop administrator with some applications of each type and state$/) do
+  require_rails_env
+  random_name = "TestWorkshopAdmin" + SecureRandom.hex[0..9]
```

Comment:
If you want a 10-char random string, do `SecureRandom.hex(10)`. This clipping of a default 16-char string seems unnecessary. See http://ruby-doc.org/stdlib-1.9.2/libdoc/securerandom/rdoc/SecureRandom.html#method-c-hex